### ISSUE TITLE
Make study public 

### ISSUE THREAD

**Issue by ElDeveloper** [qiime/qiime_web_app#2]
Add a user interface element to the web_app, that allows the user to make a study public.

**Comment by douginator2000** [comment]
Some additional details: Probably add a link to the study overview page (select_study_task.psp) that says something like "Make study public". This will flag the samples as being public (public column in sample table) and run the collate script.

**Comment by ElDeveloper** [comment]
- [ ] Add a link to make it private again.
- [ ] Call the script to collate the data once it's been made public.

To make a study public you need to flag all the samples of the study as public, instead of flagging the study only, though you should also flag the study as public.

---

The link to make the study public should go under the tools section `select_study_task.psp`.

**Comment by rob-knight** [comment]
Remember that all EMP studies are supposed to be public forever -- and making a study private should be an admin-only option. Also there are studies where only some of the data can be public (eg for EMP reasons).

On Feb 4, 2013, at 11:29 AM, "Yoshiki" <notifications@github.com<mailto:notifications@github.com>> wrote:
-    Add a link to make it private again.
-    Call the script to collate the data once it's been made public.

To make a study public you need to flag all the samples of the study as public, instead of flagging the study only, though you should also flag the study as public.

**Comment by ElDeveloper** [comment]
Yes thanks for pointing this out, general public should not have access to this option, only admins.
